src/gwt/src/org/rstudio/studio/client/application/ui/impl/WebApplicationHeader.java

```diff
@@ -411,6 +412,7 @@ private void initCommandsPanel(final SessionInfo sessionInfo)
          usernameLabel.setTitle(userIdentity);
          userIdentity = userIdentity.split("@")[0];
          usernameLabel.setText(userIdentity);
+         Roles.getPresentationRole().setAriaLabelProperty(usernameLabel.getElement(), "Username");
```

Comment:
I know this is duplicated because that's how the code is now, but out of curiosity is there any way that we can refactor some of the code in this file and DesktopApplicationHeader.java so that the common parts aren't duplicated? Not suggesting you do it in this PR, just a thought I had.

Reply:
Certainly; this code was added quite recently when Karl did the work to support Server Pro sessions with Desktop Pro (prior to that, the desktop never showed username in the header).

Reply:
That makes sense, thanks!